Console/Command/CakePHPUtilShell.php

```diff
@@ -0,0 +1,330 @@
+<?php
+App::uses('AppShellErrorHandler', 'Lib/Util');
+App::uses('Shell', 'Console');
+
+/**
+ * Application Shell
+ *
+ * Add your application-wide methods in the class below, your shells
+ * will inherit them.
+ *
+ * @package       app.Console.Command
+ *
+ * @property MailTask $Mail
+ */
+class CakePHPUtilShell extends Shell {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "CakePHPUtilShell" has 316 lines, which is greater than the 200 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2042)